modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,39 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+# ...
+spec:
+# ...
+  storage:
+    schemas: # <1>
+    - effectiveDate: "<yyyy>-<mm>-<past_dd>" # <2>
+      version: v12 # <3>
+    - effectiveDate: "<yyyy>-<mm>-<future_dd>" # <4>
+      version: v13 # <5>
+# ...
+----
+<1> Upgrading to the {clo} 5.9 or later without the `v13` schema version in the `LokiStack` custom resource will result in a web console warning.
+<2> Keep the previously set date value.
+<3> Later, after the `effectiveDate` date of the `v13` schema version, you can remove the `v12` schema version.
+<4> Set a future date, such as tomorrow's date, which has not yet started in the Coordinated Universal Time (UTC) time zone.
+<5> Add the `v13` schema version under the `v12` schema version. 
```

Comment:
I'd delete the points 4 and 5 to simplify them and I'll write only <3> where it could be

<3> Add the new schema definition after the current being `effectiveDate` as a future date, such as tomorrow's date, which has not yet started in the Coordinated Universal Time (UTC) time zone and  `v13` as schema `version` to be used.

I shouldn't mention anything about deleting the old in the future as it's not needed and in case that someone had a misunderstanding, the access to the data would be lost, something that we don't desire

Reply:
I've deleted point 5.

However, are you sure that you want me to delete point 4?
The reason I'm asking is because I added it in response to your comment in https://github.com/openshift/openshift-docs/pull/85726#discussion_r1869178626.

Reply:
I need to apologize, let's forget about the "numbers". Really, I was proposing to merge the these point 4 and 5 to be like only one with a content similar to the one below where it continues being explained the field `effectiveDate`  as you were writing:

`Add the new schema definition after the current existing one being the value of the `effectiveDate` as a future date, such as tomorrow's date, which has not yet started in the Coordinated Universal Time (UTC) time zone and the value of the schema `version` v13.

Reply:
Thank you for the explanation. I've rewritten those lines.